jenkins-slaves/jenkins-slave-ansible-stacks/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM registry.access.redhat.com/openshift3/jenkins-slave-base-rhel7
+
+MAINTAINER Johnathan Kupferer <jkupfere@redhat.com>
+
+LABEL com.redhat.component="jenkins-slave-ansible-stacks" \
+      name="jenkins-slave-ansible-stacks" \
+      architecture="x86_64" \
+      io.k8s.display-name="Jenkins Slave Ansible Stacks" \
+      io.k8s.description="Ansible and ansible-stacks on top of the jenkins slave base image" \
+      io.openshift.tags="openshift,jenkins,slave,copy"
+
+USER root
+
+RUN yum --enablerepo rhel-7-server-ose-3.3-rpms install -y \
```

Comment:
Can we add an environment variable for the repository version. This will allow for an environment variable to be defined in the BuildConfig within the template to override the default value for versions other than 3.3